app/views/govuk_publishing_components/components/docs/image_card.yml

```diff
@@ -0,0 +1,75 @@
+name: Image card
+description: Image and associated text with a link
+body: |
+  An image and an associated link, meant for use for news articles and people. Includes optional paragraph and additional links.
+
+  Note that the component is generally to be [used within a grid column](https://www.gov.uk/government/organisations/attorney-generals-office) but has no grid of its own, so the examples below may have text that extent beyond the width of the image.
+accessibility_criteria: |
+  The component must:
+
+  - include alt text for images when present
+  - not have duplicate links for the image and the text
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      href: "/not-a-page"
+      image_src: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/62756/s300_courts-of-justice.JPG"
+      heading_text: "News headline"
+  with_a_different_heading_level:
+    description: Use a different heading level for the main link title. Defaults to H2 if not passed.
+    data:
+      href: "/really-not-a-page"
+      image_src: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/62756/s300_courts-of-justice.JPG"
+      heading_text: "I am a heading level 3"
+      heading_level: 3
+  with_more_information:
+    data:
+      href: "/also-not-a-page"
+      image_src: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/62756/s300_courts-of-justice.JPG"
+      metadata: "10 May 2018 - Press release"
+      heading_text: "Government does things"
+      description: "Following a thorough review of existing procedure, a government body has today announced that further work is necessary."
+  with_extra_links:
+    data:
+      href: "/a-page-no-just-kidding"
+      image_src: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/feature/image/62756/s300_courts-of-justice.JPG"
+      heading_text: "Some more links"
+      description: "Greater transparency across government is at the heart of our commitment to let you hold politicians and public bodies to account."
+      extra_links: [
+        {
+          text: "Single departmental plans",
+          href: "/1"
+        },
+        {
+          text: "Prime Minister's and Cabinet Office ministers' transparency publications",
+          href: "/2"
+        },
+        {
+          text: "Government transparency data",
+          href: "/3"
+        },
+      ]
+  extra_links_without_indent:
+    description: Don't indent the extra links. Used for links to people pages.
+    data:
+      href: "/government/people/jeremy-wright"
+      image_src: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/person/image/354/s216_TME_3860TME_3860_960_Jeremy_Wright.JPG"
+      metadata: "The Rt Hon"
+      heading_text: "Jeremy Wright QC MP"
```

Comment:
Perhaps use a fictitious person here, these pages might accidentally be indexed by Google / found by members of the public/parliament.

Reply:
👍